src/common/runtime/server.ts

```diff
@@ -78,8 +94,21 @@ for (let i = 0; i < sys.args.length; ++i) {
   }
 }
 
+let codeCoverage: CodeCoverageProvider | undefined = undefined;
```

Comment:
Isn't that just `CodeCoverageProvider?`

Reply:
Doesn't seem to work. ESLint wants to move the `?` to the LHS of the type, and then that fails to compile. _shrug_